mobile/lib/services/rest_api.dart

```diff
@@ -17,8 +17,8 @@ import 'package:uuid/uuid.dart';
 String addQueryParameters(Map<String, dynamic> queryParams, String url) {
   Map<String, dynamic> params = queryParams;
   params.remove("TOKEN");
-  String formattedUrl = '$url?TOKEN=${Config.airqoApiV2Token}';
```

Comment:
Removing token since we're already using headers in all the requests.

Reply:
I think the token is required for v2 endpoints. @BenjaminSsempala

Reply:
@NoahNsimbe Yes, either the token or the header but since we already add headers to all the requests, token might be unncessary

Reply:
Thanks, @BenjaminSsempala.  @Baalmart lease confirm that v2 endpoints don't need the token parameter for access control.

Reply:
@NoahNsimbe , 

1. V2 use EITHER use `token` or `JWT`
2. Most Auth Service endpoints in particular require JWTs in their code for BOTH v1 and v2. So this is one thing to take note of. 
@BenjaminSsempala